functions/public/Invoke-WPFMicrowin.ps1

```diff
@@ -117,14 +118,38 @@ public class PowerManagement {
 			if (Test-Path $driverPath)
 			{
 				Write-Host "Adding Windows Drivers image($scratchDir) drivers($driverPath) "
-				Add-WindowsDriver -Path "$scratchDir" -Recurse -Driver "$driverPath"
+				dism /English /image:$scratchDir /add-driver /driver:$driverPath /recurse | Out-Host
 			}
 			else 
 			{
 				Write-Host "Path to drivers is invalid continuing without driver injection"
 			}
 		}
 
+		if ($importDrivers)
+		{
+			Write-Host "Exporting drivers from active installation..."
+			if (Test-Path "$env:TEMP\DRV_EXPORT")
+			{
+				Remove-Item "$env:TEMP\DRV_EXPORT" -Recurse -Force
+			}
+			New-Item -Path "$env:TEMP\DRV_EXPORT" -ItemType Directory -Force
+			dism /english /online /export-driver /destination="$env:TEMP\DRV_EXPORT" | Out-Host
+			if ($?)
+			{
+				Write-Host "Adding exported drivers..."
+				dism /english /image="$scratchDir" /add-driver /driver="$env:TEMP\DRV_EXPORT" /recurse | Out-Host
+			}
+			else
+			{
+				Write-Host "Failed to export drivers. Continuing without importing them..."
+			}
+			if (Test-Path "$env:TEMP\DRV_EXPORT")
+			{
+				Remove-Item "$env:TEMP\DRV_EXPORT" -Recurse -Force
+			}
+		}
```

Comment:
Perhaps, you can import drivers to the same directory where Inject Drivers is pointing to?  These are related activities anyway, you need to inject after you import...

Reply:
The thing is that this doesn't depend on the directory chosen in that option. I can work on that but leave `$env:TEMP\DRV_EXPORT` as a fallback if that option is enabled and a directory hasn't been specified

Reply:
The only problem is that I'm not good at modifying XAML code, as I'm more used to WinForms

Reply:
Already implemented to some extent by the change below